lambdaclass/mirra_backend
# Update code for loadtests

## Motivation

Loadtests weren't working with current code, several changes made in arena broke the loadtests app.

## Summary of changes

- [Add new OVERRIDE_JWT env var to avoid validation for loadtest's bots](https://github.com/lambdaclass/mirra_backend/commit/54cf5e2aa9eb23a7139796059bf6ef925ef205d2)
- [Update loadtest ws message handler due to changes in messages](https://github.com/lambdaclass/mirra_backend/commit/f858df2f775415c8f07286113d1bcf52127ceaa2)

## How to test it?

Run some loadtests and see if it works. Remember to set the new env var.
You can follow loadtest's README to do so.

## Checklist
